2-copy-of-code/lesson-14/styles/pages/checkout/checkout.css

```diff
@@ -212,4 +212,17 @@
 
 .quantity-input {
   width: 30px;
+  display: none;
+}
+
+.save-quantity-link {
+  display: none;
+}
+
+.is-editing-quantity .quantity-input {
+  display: initial;
+}
+
+.is-editing-quantity .save-quantity-link {
```

Comment:
`.is-editing-quantity.quantity-input,
.is-editing-quantity.save-quantity-link {
  display: initial;
}
`

Reply:
I put them together.Is that fine?

Reply:
```
.is-editing-quantity  {
  .quantity-input,
  .save-quantity-link {
    display: initial;
  }
}
```
You can short it like this bro.

Reply:
it resets the property back to its default value